suricata/update/main.py

```diff
@@ -216,7 +221,7 @@ def run(self, url=None):
                 files.update(fetched)
             except URLError as err:
                 url = url[0] if isinstance(url, tuple) else url
-                logger.error("Failed to fetch %s: %s", url, err)
+                logger.error("Failed to fetch {}: {}".format(url, err))
```

Comment:
This is the error line that is hit if we get a 404.. So a strict check needs to be done here.